src/engine/LoadingScreen.cs

```diff
@@ -78,22 +77,7 @@ public string LoadingDescription
         }
     }
 
-    public string Tip
-    {
-        get => tip;
-        set
-        {
-            if (tip == value)
-                return;
-
-            tip = value;
-
-            if (tipLabel != null)
-            {
-                UpdateTip();
-            }
-        }
-    }
+    public string Tip => TranslationServer.Translate("LOADING_TIP");
```

Comment:
Please keep the functionality in for changing the tip, it will be used at some point once we have longer loading screens.